core/tests/ledger_cleanup.rs

```diff
@@ -437,18 +437,6 @@ mod tests {
                 }
             }
 
-            if stop_size_bytes > 0 {
```

Comment:
This code block is a duplication from [line 428-438](https://github.com/solana-labs/solana/blob/4f545d2f1be35c8ea9399897ab43a25432cfb1ff/core/tests/ledger_cleanup.rs#L428-L438) and should be removed.